src/ar/arabshentai/src/eu/kanade/tachiyomi/extension/ar/arabshentai/ArabsHentai.kt

```diff
@@ -131,20 +132,16 @@ class ArabsHentai : ParsedHttpSource() {
         }
 
     // ============================== Chapters ==============================
-    override fun chapterListSelector() = "#chapter-list a[href*='/manga/'], .oneshot-reader .images .image-item a[href$='manga-paged=1']"
+    override fun chapterListSelector() = "#chapter-list a[href*='/manga/']"
 
     override fun chapterFromElement(element: Element) =
         SChapter.create().apply {
             val url = element.attr("href")
-            if (url.contains("style=paged")) {
-                setUrlWithoutDomain(url.substringBeforeLast("?"))
-                name = "ونشوت"
-                date_upload = 0L
-            } else {
-                name = element.select(".chapternum").text()
-                date_upload = element.select(".chapterdate").text().parseChapterDate()
-                setUrlWithoutDomain(url)
-            }
+            setUrlWithoutDomain(url)
+
+            name = element.selectFirst(".chapternum")?.text() ?: url.substringAfterLast("/").removeSuffix("/")
+
+            date_upload = element.selectFirst(".chapterdate")?.text().parseChapterDate() ?: 0L
```

Comment:
this can't be null (?:) since the function is returning `Long` instead of `Long?`

Reply:
same here

Reply:
??